### ISSUE TITLE
Suggestion for allowing cleaner interactivity with the bot

### ISSUE THREAD

**Issue by F1r3w477** [sguo1123/SMMOBot-Teddybear#37]
SimpleMMO ID/Username: ZestyChan (215516)
Type Of Issue: Suggestion

When the bot is replying to a message from a user, the response could be added to an embed message that is only visible to the user messaging the bot. This would decrease chat spam when people are using the bot.

Additionally, once the bot completes the command, you could remove the message sent by the user for the command, this would further reduce the amount of spam from using/interacting with the bot.

Current functionality:
![image](https://github.com/sguo1123/SMMOBot-Teddybear/assets/9701525/e55c5052-d593-4cd6-ac5f-cdaaf75763f2)

**Comment by F1r3w477** [comment]
I'm assuming this is in discord.py, so [this link](https://discordpy.readthedocs.io/en/stable/api.html?highlight=ephemeral#discord.MessageFlags.ephemeral) should be how to make the messaged only visible to the user.

edit:

tldr; await interaction.response.send_message(content=content, ephemeral=True) -- Something like this

**Comment by HugTeddy** [comment]
Definitely something I am rolling through implementing when updating various commands. Removing messages will also be an option however I don't require permissions to edit messages so I will need to check before deleting.
